src/components/Modal/Modal.tsx

```diff
@@ -46,12 +50,13 @@ export const Modal: React.FC<ModalProps> = ({
         ></textarea>
         <div className="modal-form__buttons">
           <Button
-            text="Create"
+            text={type === 'create' ? 'Create' : 'Edit'}
+            type={ButtonTypes.SUBMIT}
             className="button button--blue"
-            onClick={createTodo}
           />
           <Button
             text="Cancel"
+            type={ButtonTypes.BUTTON}
```

Comment:
Add a default type value for the button.

Reply:
done